_posts/pl/2021-12-17-frontend-developer.md

```diff
@@ -0,0 +1,157 @@
+
+Wszystko to sprawia, że Eximee to głównie wyzwanie technologiczne i poligon nowych rozwiązań testowanych u klientów w publicznie dostępnych systemach!
+
+Ale to nie wszystko, w projekcie budujemy też narzędzia graficznego projektowania procesów i wniosków, obsługi backoffice i inne. Poznaj potrzeby systemu spełniającego oczekiwania od klienta, przez obsługę backoffice, po pracę programistów oraz projektantów tworzących biznesowe rozwiązania oparte o dostarczaną przez Ciebie platformę.
+
+Twoi znajomi pewnie już korzystali z systemu który tworzysz, zapewne nie raz ;-)
+
+Być może Eximee z ~250k linii TypeScript i ~130k linii SASS frameworku to coś dla Ciebie.
+
+### Consdata OSS
+
+Gdy nabierzesz ochoty na eksperymenty, prototypu i ogólnie luźniejsze podejście, zawsze możesz zaplanować swój czas na rozwój lub budżet chapterowy na wsparcie jednego z naszych projektów open source skupionych na GitHub: [https://github.com/orgs/Consdata/repositories](https://github.com/orgs/Consdata/repositories).
+
+Tylko z nazwy warto wspomnieć o:
+
+- [kouncil](https://github.com/Consdata/kouncil) - ui do Kafki zbudowany na bazie doświadczeń naszych i naszych klientów,
+- [sonarqube-companion](https://github.com/Consdata/sonarqube-companion) - agregat naruszeń SonarQube rozumiejący strukturę zespołów i śledzący trendy analizy statycznej,
+- inne narzędzia jak logger, aplikacje intranetowe do newsletterów, rezerwacji parkingu, czy śledzenia feedbacku i zbierania ankiet.
+
+W Open Source nie ograniczamy się żadnymi wytycznymi, to nasze poligony. Firebase? Angular czy React? Node czy Java? Spring czy Quarkus? Nie ma znaczenia! Zrealizuj funkcjonalność i powiedz innym jak poszło!
+
+## Dołączysz?
+
+Czujesz, że frontend to może być miejsce dla Ciebie? Zainteresował Cię któryś z naszych projektów? Czujesz, że podołasz? Chciałbyś uczestniczyć w wyznaczaniu architektury i projektowaniu aplikacji web z szerokiego portoflio?
+
+Sprawdź nasze oferty, u nas każdy znajdzie coś dla siebie ;-)
+
+Prowadzimy obecnie rekrutację na stanowisko Frontend Developer. Poszukujemy osób biegłych w tworzeniu aplikacji web, które pomogą nam wyznaczyć dalszy kierunek rozwoju i rozpędzić toczące się już zmiany w naszych aplikacjach. Kluczowa jest dla nas wiedza z poziomu projektowania aplikacji, wierzymy, że braki w tworzeniu komponentów czy jedynie ogólne wyczucie "tych backendów" to tematy, nad którymi możemy już pracować razem.
+
+Jeśli tylko czujesz się na siłach, a w backendzie chociaż "wiesz co piszczy", to nie czekaj dłużej, daj mi się zaprosić na rozmowę rekrutacyjną ;)
```

Comment:
z tego z dania w artykule bym chyba zrezygnowała, to bardziej do posta na SM dla HR/można zamienić na:
_Jeszcze ciepłą ofertę znajdziesz tutaj:_

Reply:
ważne dla mnie jest podkreślenie, że na rozmowę można trafić ze mną, wprost autorem posta i osobą o przedstawianym stanowisku.

Reply:
ok